Utils/ListUtils.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+
+namespace TheBereftSouls.Utils
+{
+    public class ListUtils
+    {
+        public static void AddSomeElements(HashSet<int> list, HashSet<int> insert)
+        {
+            foreach (int element in insert)
+                list.Add(element);
+        }
```

Comment:
Utilities like these are redundant as C# has `Union` and `UnionWith` as available methods for `HashSet<T>`.

For more info:
- [Detailed comparison between `Union` and `UnionWith` for `HashSet<T>`](https://stackoverflow.com/a/47327133 "Stack Overflow a/47327133")
- [C# `IEnumerable.Union`](https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.union "learn.microsoft.com | system.linq.enumerable.union") *(This also applies for `List<T>`)*

Reply:
idk how use union for that, can you give me an example?